content/en/docs/concepts/containers/images.md

```diff
@@ -33,17 +33,35 @@ Container images are usually given a name such as `pause`, `example/mycontainer`
 Images can also include a registry hostname; for example: `fictional.registry.example/imagename`,
 and possibly a port number as well; for example: `fictional.registry.example:10443/imagename`.
 
-If you don't specify a registry hostname, Kubernetes assumes that you mean the Docker public registry.
+If you don't specify a registry hostname, Kubernetes assumes that you mean the [Docker public registry](https://hub.docker.com/).
+You can change this behaviour by setting default image registry in [container runtime](/docs/setup/production-environment/container-runtimes/) configuration.
```

Comment:
Do we need also to mention possible usage of MutationWebhook for images here? Looks like it is also a common solution to change the default registry

 (I use it in combination with ECR and self-written webhook. Also, I've seen an implementation for kyverno/OPA gatekeeper)

Reply:
It's good to mention, if one to use different image registry than default one.